VocaDbWeb/wwwroot/Scripts/ViewModels/Search/SongSearchViewModel.ts

```diff
@@ -198,6 +206,38 @@ import Decimal from 'decimal.js-light';
 				}
 			});
 
+			this.minLengthFormatted = ko.computed({
+				read: () => {
+					return DateTimeHelper.formatFromSeconds(this.minLength());
+				},
+				write: (value: string) => {
+					var parts = value.split(":");
+					if (parts.length == 2 && parseInt(parts[0], 10) != NaN && parseInt(parts[1], 10) != NaN) {

```

Comment:
You should really extract this to a helper method (also from the song edit page). Or even better, Knockout binding handler. But if you don't want to go to deep with Knockout, then just the helper method.